config/jobs/kubernetes/sig-node/sig-node-presubmit.yaml

```diff
@@ -216,3 +216,42 @@ presubmits:
           env:
             - name: GOPATH
               value: /go
+  - name: pull-kubernetes-node-crio-1.18-e2e
+    skip_branches:
+    - release-\d+\.\d+  # per-release image
+    annotations:
+      fork-per-release: "true"
+      testgrid-alert-stale-results-hours: "24"
+      testgrid-create-test-group: "true"
+      testgrid-num-failures-to-alert: "10"
+    always_run: false
```

Comment:
This means in the first place we select single PRs and trigger them manually there, right?

Reply:
Yes